build/.goreleaser.yml

```diff
@@ -30,17 +30,19 @@ builds:
 
 nfpms:
   - id: linux
-    builds:
-      - nri-jmx-nix
     file_name_template: "{{ .ProjectName }}_{{ .Version }}-1_{{ .Arch }}"
     vendor: "New Relic, Inc."
     homepage: "https://www.newrelic.com/infrastructure"
     maintainer: "New Relic Infrastructure Team <infrastructure-eng@newrelic.com>"
     description: "New Relic Infrastructure jmx Integration extend the core New Relic\nInfrastructure agent's capabilities to allow you to collect metric and\nlive state data from jmx components."
     license: "https://newrelic.com/terms (also see LICENSE installed with this package)"
-    
+
+    builds:
+      - nri-jmx-nix
+
     dependencies:
       - "newrelic-infra"
+      - "nrjmx"
```

Comment:
Seems to collide with https://github.com/newrelic/nri-jmx/pull/79, I'd wait until that's merged

Reply:
Yes, I am aware, I will wait until that is merged to rebase and get the >= 1.5.3

Reply:
I have rebased this branch and now it has the changed done by Cristian.